src/README.md

```diff
@@ -1,6 +1,7 @@
 # Pulsed Power Monitoring
```

Comment:
OpenCMW - How to run
add 
- run ./opencmw_worker/build/InferenceTool 
- information of data files from the integrators